sdk/test-utils/perfstress/karma.conf.js

```diff
@@ -0,0 +1,83 @@
+// https://github.com/karma-runner/karma-chrome-launcher
```

Comment:
I'm not currently using this file, but it will be relevant for: https://github.com/azure/azure-sdk-for-js/issues/8064